build.sbt

```diff
@@ -13,42 +13,86 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 import sbt.Keys._
 import sbt._
-import uk.gov.hmrc.DefaultBuildSettings.targetJvm
-import uk.gov.hmrc.versioning.SbtGitVersioning
-import uk.gov.hmrc.versioning.SbtGitVersioning.autoImport.majorVersion
-import uk.gov.hmrc.{SbtArtifactory, SbtAutoBuildPlugin}
+val name = "http-metrics"
+
+val scala2_12 = "2.12.12"
+
+// Disable multiple project tests running at the same time: https://stackoverflow.com/questions/11899723/how-to-turn-off-parallel-execution-of-tests-for-multi-project-builds
+// TODO: restrict parallelExecution to tests only (the obvious way to do this using Test scope does not seem to work correctly)
+parallelExecution in Global := false
+
+val silencerVersion = "1.7.1"
 
-lazy val deps: Seq[ModuleID] = compile ++ test
+lazy val commonSettings = Seq(
+  organization := "uk.gov.hmrc",
+  majorVersion := 2,
+  scalaVersion := scala2_12,
+  makePublicallyAvailableOnBintray := true,
+  resolvers := Seq(
+    Resolver.bintrayRepo("hmrc", "releases"),
+    Resolver.typesafeRepo("releases")
+  ),
+  scalacOptions ++= Seq("-feature"),
+  // libraryDependencies ++= Seq(
```

Comment:
Don't we need this?

Reply:
Yes we do.  But I wanted to see the errors first.... Lol.